app/modules/asset_groups/parameters.py

```diff
@@ -160,6 +160,40 @@ def replace(cls, obj, field, value, state):
             cls.validate_asset_references(obj, value)
             obj.sighting_config[field] = value
             ret_val = True
+
+        elif field == 'timeSpecificity':
+            log.debug('updating time specificity')
+            from app.modules.complex_date_time.models import ComplexDateTime
+
+            # If no time in the config, using a valid time here allows the specificity to be set
+            # This is a reasonably safe assumption as (1) This should never happen and (2) if we blocked this,
+            # it would make it actually difficult to fix and (3) The FE should always send the time anyway.
+            timeData = {
+                'time': obj.sighting_config.get('time', '2020-05-01T00:00:00+01:00'),
```

Comment:
subtle May Day reference?

Reply:
I wonder if it might be safer to pick a more obviously-wrong time like 1970-01-01, admitting this shouldn't happen anyway.

Reply:
Lol, Jon asked the same question. No it was arbitrary. 
Given that this should never happen (outside of wacky tests) and if it did, the patch will contain time and timespecificity together and will fix it, I didn't give too much thought to the value other than.. make it work.